draft-add-requirements.md

```diff
@@ -232,23 +277,22 @@ this threat model may have limited value.
 This section lists requirements that flow from the above sections.
 
 | Requirement | Description |
-| R1.1 | Discovery MUST provide a local network the ability to announce to clients a set of, or absence of, equivalent resolvers. |
-| R1.2 | Discovery MUST provide a resolver the ability to announce to clients a set of, or absence of, equivalent resolvers. |
+| R1.1 | Discovery SHOULD provide a local network the ability to announce to clients a set of, or absence of, designated resolvers. |
+| R1.2 | Discovery SHOULD provide a resolver the ability to announce to clients a set of, or absence of, designated resolvers. |
 | R1.3 | Discovery MUST support at least one encrypted DNS protocol. |
 | R1.4 | Discovery SHOULD support all standardised encrypted DNS protocols. |
-| R2.1 | Networks MUST be able to announce one or more equivalent encrypted DNS resolvers using existing mechanisms such as DHCPv4, DHCPv6, IPv6 Router Advertisement, and the Point-to-Point Protocol. |
-| R2.2 | The format for resolver information MUST be specified such that provisioning mechanisms defined outside of the IETF can advertise encrypted DNS resolvers. |
+| R2.1 | Networks SHOULD be able to announce one or more designated encrypted DNS resolvers using existing mechanisms such as DHCPv4, DHCPv6, IPv6 Router Advertisement, and the Point-to-Point Protocol. |
+| R2.2 | The format for resolver designation SHOULD be specified such that provisioning mechanisms defined outside of the IETF can advertise encrypted DNS resolvers. |
+| R2.3 | This format SHOULD convey, at minimum, the information the client needs to make contact with each designated resolver. |
+| R2.4 | This format MAY convey more information. |
 | R3.1 | When discovery is instantiated from a resolver (R1.2), that resolver MAY be encrypted or not. |
-| R3.2 | When discovery is instantiated from a resolver (R1.2), that resolver MAY be locally or globally reachable. Both options MUST be supported. |
-| R4.1 | In a home network use case, if the local network forwarder does not offer encrypted DNS service, the ISP's encrypted DNS server information MUST be retrievable via a query sent to a local network forwarder. |
-| R4.2 | Encrypted DNS server discovery MUST NOT require any changes to DNS forwarders hosted on non-upgradable legacy network devices. |
+| R3.2 | When discovery is instantiated from a resolver (R1.2), that resolver MAY be locally or globally reachable. Both options SHOULD be supported. |
+| R4.1 | In a home network use case, if the local network resolver is a forwarder that does not offer encrypted DNS service, an upstream encrypted resolver SHOULD be retrievable via queries sent to that forwarder. |
```

Comment:
I would drop "In a home network use case" ?
Local forwarder could be used in SMB, SOHO and branch offices with split tunnel connection.